ProjectList.js

```diff
@@ -43,7 +44,10 @@ var ProjectList = React.createClass({
         });
       })
       .catch((error) => {
-        console.error(error);
+        Alert.alert(
+          'Error',
+          'The following error occurred.  Please close down Zooniverse and try again.  If it persists please notify us.  \n\n' + error,
```

Comment:
Yeah it's just twice for extra space, I think it reads a slightly better (and btw, if you have any ideas for wording changes, feel free to add them!)

![image](https://cloud.githubusercontent.com/assets/1033477/18289933/1d7d92b6-7447-11e6-9e08-c44fc66fb18a.png)

vs. 

![image](https://cloud.githubusercontent.com/assets/1033477/18289954/2f4ac450-7447-11e6-97d2-39c306ff241d.png)